### ISSUE TITLE
Out of range: End of sequence

### ISSUE THREAD

**Issue by longpeng2008** [dikatok/fast-deep-matting#2]
Out of range: End of sequence error when come to epoch 55000, I can reproduce the bug

**Comment by dikatok** [comment]
Did you make any changes? That error should already be handled by MonitoredTrainingSession

**Comment by longpeng2008** [comment]
no, I don't know why, can you reproduce the result of the paper using the image matting data you send to me, do you finetune the model or just  train from scratch

**Comment by dikatok** [comment]
I am getting around 92% and 85% IOU for my training and validation set, the main difficulty is when it is fed portrait in the wild the results are not convincing, most likely because the lack of variation in the dataset. Currently I am planning to use COCO and/or VOC dataset as well in addition to the portrait dataset. You should probably check out my pix2pix matting repository if interested, it is using u-net encoder-decoder network which has better results in my experiments but it is computationally expensive to fit in client side like mobile phone.

**Comment by dikatok** [comment]
And I will be training from scratch when I add the new dataset since the data distribution is not exactly the same.

**Comment by longpeng2008** [comment]
so for now you finetuned your model using the size 128*128？  can you provide your email for convenience

**Comment by dikatok** [comment]
yes, I was finetuning my model with the same size and sure holy.arc.64633@gmail.com

**Comment by longpeng2008** [comment]
thanks, I reproduced your results by your data, keep contact

**Comment by 24rochak** [comment]
I am using your model for my own data but not able to get desired result. Can u share the data which you used so I can verify if there is class imbalance in my dataset. Also is there a pretrained model available ??
